### PR TITLE
Update Boards release process

### PR DESCRIPTION
**Pull request by amyblais**
Note for reviewers:

 - If there are any other resources/links related to the Boards release process that could be added here, we could consolidate everything here in one place.
 - Please help check that all the details, including the Release Timeline are up-to-date.
 - Please help check if there are any items that could be added to the Boards' team's release playbook.

### CODE REVIEW

File: operations/research-and-development/product/release-process/boards-release.md [hunk 1]
```diff
@@ -1,35 +1,34 @@
-# Focalboard Release Process
+# Boards Release Process
 
-This document outlines the Focalboard release process for the Mattermost core team. Our release process draws from many best practice Agile, Scrum, and Software Development Lifecycle methodologies.
+This document outlines the Boards release process for the Mattermost core team. Our release process draws from many best practice Agile, Scrum, and Software Development Lifecycle methodologies.
 
 ## Release Timeline
 
-Focalboard release schedule for July and August is outlined here: [https://docs.google.com/spreadsheets/d/1nMrpSyR8PSEkHuQl\_oJ-bAz1\_qpEiGhgF4HdDX1Uf4Q/edit\#gid=0](https://docs.google.com/spreadsheets/d/1nMrpSyR8PSEkHuQl_oJ-bAz1_qpEiGhgF4HdDX1Uf4Q/edit#gid=0).
+Boards release schedule is outlined here:
```
Comment (amyblais) [inline operations/research-and-development/product/release-process/boards-release.md]: Let me know if this is outlined somewhere.

Comment (chenilim) [reply operations/research-and-development/product/release-process/boards-release.md]: The Boards monthly release process is currently captured in the [Monthly Release Playbook](https://community-daily.mattermost.com/playbooks/playbooks/xz7uzn4ui7fpice9xn3dyaeu6h/preview).

As for the key dates each month, @wuwinson will own those going forward, and they will align with the Mattermost dates [per this thread](https://community-daily.mattermost.com/core/pl/dx7e8716sig75gu1qzj7rsarzr).

File: operations/research-and-development/product/release-process/boards-release.md [hunk 2]
```diff
@@ -1,35 +1,34 @@
...
 
 ## Release Timeline
 
-Focalboard release schedule for July and August is outlined here: [https://docs.google.com/spreadsheets/d/1nMrpSyR8PSEkHuQl\_oJ-bAz1\_qpEiGhgF4HdDX1Uf4Q/edit\#gid=0](https://docs.google.com/spreadsheets/d/1nMrpSyR8PSEkHuQl_oJ-bAz1_qpEiGhgF4HdDX1Uf4Q/edit#gid=0).
+Boards release schedule is outlined here: 
 
-The release that will be included in the next self-managed release pre-packaged version will generally be the latest Focalboard Personal Edition release from the prior month. The deadline to get the pre-packaged version to the self-managed release is by self-managed Code Freeze day. The pre-packaged version is tested a few days prior to self-managed Code Freeze day.
+The release that will be included in the next self-managed release pre-packaged version will generally be the latest Boards Personal Edition release from the prior month. The deadline to get the pre-packaged version to the self-managed release is by self-managed Code Freeze day. The pre-packaged version is tested a few days prior to self-managed Code Freeze day.
```
Comment (chenilim) [inline operations/research-and-development/product/release-process/boards-release.md]: This is still technically "Focalboard Personal Edition". "Boards" generally only refers to Boards that ships with Mattermost.

**Review by chenilim — CHANGES_REQUESTED** [review]
Should be "Focalboard Personal Edition". Other than that looks good. Thanks!

**Review by sbishel — DISMISSED** [review]

File: operations/research-and-development/product/release-process/boards-release.md [hunk 3]
```diff
@@ -1,35 +1,34 @@
...
-Focalboard release schedule for July and August is outlined here: [https://docs.google.com/spreadsheets/d/1nMrpSyR8PSEkHuQl\_oJ-bAz1\_qpEiGhgF4HdDX1Uf4Q/edit\#gid=0](https://docs.google.com/spreadsheets/d/1nMrpSyR8PSEkHuQl_oJ-bAz1_qpEiGhgF4HdDX1Uf4Q/edit#gid=0).
+Boards release schedule is outlined here: https://docs.google.com/spreadsheets/d/1nMrpSyR8PSEkHuQl_oJ-bAz1_qpEiGhgF4HdDX1Uf4Q/edit#gid=1279713913.
 
 The release that will be included in the next self-managed release pre-packaged version will generally be the latest Focalboard Personal Edition release from the prior month. The deadline to get the pre-packaged version to the self-managed release is by self-managed Code Freeze day. The pre-packaged version is tested a few days prior to self-managed Code Freeze day.
 
 ## Notes
 
-* **TBD:** Critical bugs are flagged and fixed before releasing, with quick dot releases completed as needed.
-* **TBD:** Incident Collaboration Playbook is used to run releases.
+* Critical bugs are flagged and fixed before releasing, with quick dot releases completed as needed.
```
Comment (dubdubs01) [inline operations/research-and-development/product/release-process/boards-release.md:13]: @amyblais Can potentially consider adding this bullet below line 13:
`Documentation on filing a bug: https://handbook.mattermost.com/contributors/contributors/ways-to-contribute#report-a-boards-bug`

**Review by dubdubs01 — DISMISSED** [review]
Looks good to me!

**Review by chenilim — DISMISSED** [review]

File: operations/research-and-development/product/release-process/boards-release.md [hunk 4]
```diff
@@ -1,35 +1,35 @@
...
+* Documentation on filing a bug: https://handbook.mattermost.com/contributors/contributors/ways-to-contribute#report-a-boards-bug.
+* A Playbook is used to run releases: https://community.mattermost.com/playbooks/playbooks/xycc3rjwd3n95n1e1yqxp9iira/preview.
```
Comment (justinegeffen) [inline operations/research-and-development/product/release-process/boards-release.md]:
```suggestion
* A playbook is used to run releases: https://community.mattermost.com/playbooks/playbooks/xycc3rjwd3n95n1e1yqxp9iira/preview.
```

**Review by justinegeffen — APPROVED** [review]

**Pull request merged**